pkg/services/ngalert/metrics/remote_alertmanager.go

```diff
@@ -0,0 +1,36 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/weaveworks/common/instrument"
+)
+
+type RemoteAlertmanager struct {
+	HTTPRequestsTotal    *prometheus.CounterVec
+	HTTPRequestsFailed   *prometheus.CounterVec
+	HTTPRequestsDuration *instrument.HistogramCollector
+}
+
+func NewRemoteAlertmanagerMetrics(r prometheus.Registerer, subsystem string) *RemoteAlertmanager {
+	return &RemoteAlertmanager{
+		HTTPRequestsTotal: promauto.With(r).NewCounterVec(prometheus.CounterOpts{
+			Namespace: Namespace,
+			Subsystem: subsystem,
+			Name:      "remote_alertmanager_http_requests_total",
+			Help:      "Number of HTTP requests sent to the remote Alertmanager.",
+		}, []string{"method", "path", "status_code"}),
+		HTTPRequestsFailed: promauto.With(r).NewCounterVec(prometheus.CounterOpts{
+			Namespace: Namespace,
+			Subsystem: subsystem,
+			Name:      "remote_alertmanager_http_requests_failed",
+			Help:      "Number of failed attempts to send an HTTP request to the remote Alertmanager.",
+		}, []string{"method", "path"}),
```

Comment:
We don't need these - we can get all this information by just using the histogram.